dune-project

```diff
@@ -32,10 +34,9 @@ JavaScript via [js_of_ocaml](http://ocsigen.org/js_of_ocaml)."))
  (depends
   (ocaml (>= 4.08.0))
   (gitlab (= :version))
-  (cmdliner (>= 1.1.0))
+  (cmdliner (>= 2.0.0))
```

Comment:
This conflicts with ocamlformat, which doesn't support cmdliner 2.0 just yet. https://github.com/ocaml-ppx/ocamlformat/issues/2726